test/WebJobs.Script.Tests/Workers/Rpc/GrpcWorkerChannelTests.cs

```diff
@@ -939,6 +939,23 @@ public async Task ReceivesInboundEvent_FunctionLoadResponse()
             Assert.True(traces.Any(m => string.Equals(m.FormattedMessage, "Received FunctionLoadResponse for function: 'js1' with functionId: 'TestFunctionId1'.")), "FunctionLoadResponse TestFunctionId1");
         }
 
+        [Fact]
+        public async Task ReceivesInboundEvent_Error_FunctionLoadResponse()
+        {
+            await CreateDefaultWorkerChannel();
+            var functionMetadatas = GetTestFunctionsList("node");
+            _workerChannel.SetupFunctionInvocationBuffers(functionMetadatas);
+            _testFunctionRpcService.OnMessage(StreamingMessage.ContentOneofCase.FunctionLoadRequest,
+                _ => _testFunctionRpcService.PublishSystemErrorFunctionLoadResponseEvent("TestFunctionId1", "abc AccountKey== "));
```

Comment:
We will lose all content that follows a secret. For instance, "abc Accountkey==<key>, some useful information" will be logged as "abc [Hidden Credential]". Is there any important information that might be lost in this process?

Reply:
It will not replace the entire content after secret with hidden credential. If the input is `abc Accountkey=<value> some text` then the replacement will just be till "value" i.e. `<abc [Hidden Credential] some text`. For more test cases, refer [SanitizerTests.cs](https://github.com/Azure/azure-functions-host/blob/6bccacfed716d1ec778c94fc3513cc351f497f14/test/WebJobs.Script.Tests/SanitizerTests.cs#L44)